onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -63,7 +63,10 @@ Module Name:
 #endif
 #if defined(__wasm__)
 #define MLAS_TARGET_WASM
-#if defined(__wasm_simd128__)
+#if defined(__wasm_relaxed_simd__)
+#define MLAS_TARGET_WASM_RELAXED_SIMD
+#define MLAS_TARGET_WASM_SIMD
+#elif defined(__wasm_simd128__)
```

Comment:
will both `__wasm_simd128__` and `__wasm_relaxed_simd__` be defined? 

if so, is it a redefinition of `MLAS_TARGET_WASM_SIMD`?

Reply:
I use `#elif` here, so the `__wasm_simd128__` check will be ignored if `__wasm_relaxed_simd__` is defined. I guess there's no redefinition here.